workflow/tests/base_results/results.csv

```diff
@@ -292,6 +296,8 @@ base-hvac-room-ac-only-33percent.xml,31.889,31.889,31.889,31.889,0.0,0.0,0.0,0.0
 base-hvac-room-ac-only-detailed-setpoints.xml,33.276,33.276,33.276,33.276,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,2.802,0.0,9.198,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,5.165,9.234,0.65,0.0,0.0,0.0,0.0,0.0,3083.6,0.0,9.504,0.0,2.821,2.69,0.377,4.307,0.39,-4.834,0.0,0.0,1.572,-0.219,3.293,0.0,0.552,0.0,0.0,-11.285,0.0,-0.093,-0.594,-0.07,2.489,-0.063,9.532,0.0,0.0,-6.68,-0.06,-1.274,-7.536,-0.19,0.0,0.0,10.039,1354.8,997.6,11399.6,2615.8,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
 base-hvac-room-ac-only.xml,34.642,34.642,34.642,34.642,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,4.161,0.0,9.205,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,7.919,9.234,0.659,0.0,0.0,0.0,0.0,0.0,3352.8,0.0,10.793,0.0,2.788,2.655,0.372,4.288,0.381,-4.945,0.0,0.0,1.82,-0.203,3.246,0.0,0.546,0.0,0.0,-11.287,0.0,0.056,-0.433,-0.047,2.915,-0.023,10.02,0.0,0.0,-6.132,-0.058,-1.147,-6.849,-0.16,0.0,0.0,10.046,1354.8,997.6,11399.6,2615.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
+base-hvac-room-ac-with-heating.xml,49.374,49.374,49.374,49.374,0.0,0.0,0.0,0.0,0.0,0.0,14.666,0.0,0.0,4.265,0.0,9.166,0.0,0.0,4.51,0.0,0.334,0.0,0.0,0.0,0.0,2.22,0.0,0.0,0.319,0.365,1.513,1.528,0.0,2.114,8.374,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,0.0,14.653,8.144,9.234,0.616,0.0,0.0,0.0,0.0,5610.2,3234.7,15.871,10.986,0.0,3.744,3.646,0.513,7.128,0.632,-2.014,0.0,0.0,6.647,-0.118,4.807,0.0,0.729,0.0,0.0,-11.389,0.0,0.075,-0.422,-0.046,2.936,-0.016,9.98,0.0,0.0,-5.631,-0.056,-1.121,-7.045,-0.158,0.0,0.0,9.901,1354.8,997.6,11399.6,2615.9,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0
```

Comment:
@yzhou601 Why does base-hvac-room-ac-with-heating.xml have different energy use than base-hvac-ptac-with-heating.xml? Same question for base-hvac-room-ac-only.xml vs base-hvac-ptac.xml.

Reply:
They're using different EER values, Room AC: 8.5, PTAC: 10.7 (So ptac test home is consuming less energy than room ac test home.), while pthp and room hp are configured with the same efficiencies. I can change the ptac and room ac to be of the same efficiencies if needed.

Reply:
They're using different EER values, Room AC: 8.5, PTAC: 10.7 (So ptac test home is consuming less energy than room ac test home.), while pthp and room hp are configured with the same efficiencies. I can change the ptac and room ac to be of the same efficiencies if needed.